<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rapoarte</class>
 <widget class="QDialog" name="Rapoarte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rapoarte Existente</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/rapoarte</normaloff>:/rapoarte</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Denumire</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&amp;Adaugare</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/add</normaloff>:/add</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewButton">
       <property name="text">
        <string>&amp;Vizualizare</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/rapoarte</normaloff>:/rapoarte</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="text">
        <string>&amp;Printare</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/print</normaloff>:/print</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>&amp;Editare</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/edit</normaloff>:/edit</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>&amp;Stergere</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/delete</normaloff>:/delete</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Anulare</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/cancel</normaloff>:/cancel</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="loadingLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Rapoarte</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
